public/templates/public/components/base.html

```diff
@@ -1,18 +1,22 @@
 {% load staticfiles %}
+{% load render_bundle from webpack_loader %}
 
 <!DOCTYPE html>
 <html>
 	<head>
 		<meta charset="utf-8">
 
 		<title>{% block title %}{% endblock %}</title>
+		{% render_bundle 'main' 'css' %}
 	</head>
 
 	<body>
+		<div id="root"></div>
 		{% include "./header.html" %}
 
 		{% block content %}{% endblock %}
 
 		{% include "./footer.html" %}
+		{% render_bundle 'main' 'js' %}
```

Comment:
Will there ever be any race conditions here, regarding loading when the page hasn't loaded yet? Should the JS be put inside a `<script>` in the `<head>` that executes on-page-load? Or is that taken care of somehow?

Reply:
Haha, I stuck the bundle in the footer for now as a lazy way to take care of that, to make sure the DOM is fully loaded before fetching/running any JS.

I think we can revisit this when we have our first real React components. But yeah, ideally we'd load `bundle.js` async in the `<head>`, and run it once the DOM is ready.

Reply:
🎋